### ISSUE TITLE
Android-6.0.0-beta.4 : Expression.color(int color)

### ISSUE THREAD

**Issue by emerciercontexeo** [mapbox/mapbox-gl-native#11502]
**Platform:Android**
**Mapbox SDK version:6.0.0-beta.4**

I don't know if it is a misunderstanding of Expression.color(int color) or if there is really a bug on it.

### Steps to trigger behavior

 1. Expression ex = Expression.color(0x00FF00); OU Expression ex = Expression.color(Color.GREEN);
 2. FillLayer.setProperties(PropertyFactory.fillColor(ex))

### Expected behavior

Display green polygon for this layer

### Actual behavior

A/libc: Fatal signal 6 (SIGABRT), code -6 in tid 7015 (io.mapwize.app), pid 7015 (io.mapwize.app)

Etienne

**Comment by tobrun** [comment]
For completeness, we also have a couple of color expressions as:

`rgb(int red, int green, int blue)` and `rgba(int red, int green, int blue, int alpha)`

That said I'm not able to reproduce your crash:

When I use the following expression:

```java
    Expression textColorExpression = match(get(literal(FEATURE_RANK)),
      literal(1), Expression.color(0xFF00FF00),
      literal(2),  Expression.color(Color.BLUE),
      Expression.color(0xFF00FF00)
    );
```

<img width="409" alt="screen shot 2018-03-22 at 13 01 00" src="https://user-images.githubusercontent.com/2151639/37785657-19bdc39a-2dd1-11e8-8f3c-5aff763195b3.png">

**Comment by tobrun** [comment]
@emerciercontexeo was able to reproduce with the setup shown in OP. To unblock you you can use `fillColor(rgba(0.0f, 255.0f, 0.0f, 1f))` for now, this will not crash.

**Comment by emerciercontexeo** [comment]
Yes I found other way to do it. It's why i was thinking that something goes wrong with this one.
Thanks for your answer anyway.

**Comment by tobrun** [comment]
landed with #11506
